test/exe/version_out_test.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+set -e -o pipefail
+
+source/exe/envoy-static --version | grep --fixed-strings $(cat external/envoy_api/VERSION)
```

Comment:
I kind of worry that this will be too specific to Bazel tree layout. If you want to keep it, we can see how it goes and remove later, but I don't feel it's that necessary.

Reply:
Yeah, I can now feel this is too specific for the test. Instead, I just use more loosen specification for this test at 6c21077e6 (and d4526c203). Is it near the one you thought? I'm not pretty sure what you worried about.